contracts/scripts/operator_whitelist.sh

```diff
@@ -9,8 +9,10 @@ cd "$parent_path"
 cd ../
 
```

Comment:
Export and source were not working for me. I had to add `source scripts/.env` here.